### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by Bertbk** [PandaScience/FortySecondsCV#1]
Thanks for your great template! I now use it for my résumé and it's just perfect!

As I modified your template a little bit, I was wondering if these changes could interest you and the users of the template (I can do a PR or whatever):

1. Add option for LuaLatex (not only XeTeX)
2. Profil Picture is no more mandatory
3. Following 2., peronnal information on left sidebar of the first page are mandatory and the compilation breaks down if missing one of it:
```tex
\newcommand*{\cvbirthday}[1]{\renewcommand{\cvbirthday}{#1}}
\newcommand*{\cvaddress}[1]{\renewcommand{\cvaddress}{#1}}
...
```
Instead of one command per data, I merged them in a common command `\addcvdata` for which we have to give the icon and the text:
```tex
\newcommand*{\personnaldata}{}
\newcommand{\addcvdata}[2]{\apptocmd{\personnaldata}{\circleicon{#1} & #2\\}{}{}}
```
The command `personnaltable` is then adapted obviously
4. FontAwesome is great but misses [the AcademicIcons](https://github.com/jpswalsh/academicons), which I added as a Require Package.

**Comment by PandaScience** [comment]
I'm glad you like it. I have a local version with a bunch of further changes/features lying around for almost 2 months now and couldn't find the time to review again and commit since I'm too busy with my physics code. However, I will definitely add your suggestions. You're welcome to create a pull request, then I will try to review the stuff within the next few weeks.

**Comment by Bertbk** [comment]
Thanks for your answer. I just submitted a PR.

**Comment by PandaScience** [comment]
So, I finally found some time to review your changes. Thank you very much for the PR #2! 
Here some notes:

* I really liked your idea to use `\cvphone` etc. as convenience commands as required, but under the hood use one common macro `\cvcustomdata`. (bf178877)
* Since I have no experience with LuaLaTeX I just assume that all changes will work there too, but did not test them. However, thanks for extending the class wrt. this compiler. (c51c7602, e0e3f46)
* I renamed some of your new commands for consistency, but also some my old ones. (78ec71e8)
* I wasn't really satisfied with the 'social' commands from your PR and the environment I created for them later, because they introduced too much redundant code. So I refactored / merged the old code with your ideas (bf801aa3) and the improved spacing feature (31834ef1) and came up with a common `iconlist` environment which also features some more customization options now. (60d98c65)
* Regarding some of your text formatting in the class file, e.g. for social icons, I think it is better to let the user decide how to format corresponding text. This ensures maximum customizability - which is the purpose of this re-implementation of twentysecondcv. (78ec71e8)

I also had a look at your CV at cnrs.fr and as far as I can see, with all new features you should be able to reproduce that pdf with ease - and fix the width of your github icon en passant :wink:. 
So I'm going to close this issue.
